ctapipe/image/muon/muon_diagnostic_plots.py

```diff
@@ -125,14 +124,6 @@ def plot_muon_event(event, muonparams):
 
             signals = image * clean_mask
 
-            muon_incl = np.sqrt(
-                muonparams['MuonRingParams'][idx].ring_center_x ** 2. +
-                muonparams['MuonRingParams'][idx].ring_center_y ** 2.)
-
-            muon_phi = np.arctan(
-                muonparams['MuonRingParams'][idx].ring_center_y /
-                muonparams['MuonRingParams'][idx].ring_center_x)
-
```

Comment:
are these simply not used? or is there some other reason for removing the calculation?

Reply:
Don't remove these lines, would prefer these were added to a container (e.g. add muon inclination to the muon ring parameters container)